README.md

```diff
@@ -8,20 +8,19 @@ This repository contains recipes for building binaries for Julia packages using
 
 ## Contributing
 
-To contribute a new recipe, you can either
+For detailed information about contributing, go to ["CONTRIBUTING.md"](https://github.com/JuliaPackaging/Yggdrasil/blob/master/CONTRIBUTING.md). For a quick overview, continue reading.
+
+To update the version to build for an existing recipe, simply open a PR to this repository making the required tweaks. This usually boils down to only updating the version number and the source (e.g., URL and hash for an archive, or the revision for a git repository) of the relevant `build_tarballs.jl` file, but in some cases more changes may be needed. For an example, see [this PR](https://github.com/JuliaPackaging/Yggdrasil/pull/8833). The version number should be easy to find, and the URL + hash can be found by clicking the release, clicking the final commit in the release, and copying the information from the URL bar.
 
+To contribute a new recipe, you can either
 * use `BinaryBuilder.run_wizard()`, which will automatically open a pull request to this repository after a successfull build for all requested platforms
 * Copy another build recipe using it as a template, and then open a manual pull request to this repository
 
-Yggdrasil builds the tarballs using `master` version of BinaryBuilder.jl, which requires Julia 1.3.0 or later versions.  Note that this BinaryBuilder.jl version has some differences compared to v0.1.4 and the builders generated are slightly different.  You are welcome to contribute builders written for  BinaryBuilder.jl v0.1.4, but they will likely need minor adjustements.
+Yggdrasil builds the tarballs using `master` version of BinaryBuilder.jl, which requires Julia 1.3.0 or later versions. Note that this BinaryBuilder.jl version has some differences compared to v0.1.4 and the builders generated are slightly different. You are welcome to contribute builders written for  BinaryBuilder.jl v0.1.4, but they will likely need minor adjustements.
```

Comment:
The [double spacing was there for a reason](https://en.wikipedia.org/wiki/Sentence_spacing#Computer_era) and I'd appreciate not changing it.

Reply:
I mustsay that adapting the file content to a specific editor seems like a strange way to do things. Then again, stuff must work for the main dev(s), and there is no hard in markdown, so fair enough.

Reply:
As a general rule when contributing to other projects: don't touch unrelated code to the point of your PR especially if it isn't for any good reason.  It make revision harder, it distracts from the main point, it needlessly break git bisect/blame, etc.

Reply:
Fair point. I re-added this in a later commit.